lib/bap/bap_self.ml

```diff
@@ -126,18 +132,25 @@ module Create() = struct
         | None -> value in
       value
 
-    let param converter ~default ?(docv="VAL") ?(doc="Undocumented") name =
+    let param converter ?default ?(docv="VAL") ?(doc="Undocumented") name =
       let future, promise = Future.create () in
+      let default =
+        let _, _, converter_default = converter in
```

Comment:
`converter_default` actually rings that we miss some opportunities, provided by a language. `converter.default` would be much better.